core/controllers/questions_list.py

```diff
@@ -61,12 +65,18 @@ def get(self, comma_separated_skill_ids):
         except Exception as e:
             raise self.PageNotFoundException(e)
 
-        (
-            question_summaries, merged_question_skill_links,
-            next_start_cursor) = (
-                question_services.get_displayable_question_skill_link_details(
-                    constants.NUM_QUESTIONS_PER_PAGE, skill_ids, start_cursor)
-            )
+        question_summaries, merged_question_skill_links, _ = (
```

Comment:
Why do you need get_displayable_question_skill_link_details() to return a third arg if it's not used?

Reply:
Yeah, if you're not using `cursor` anymore, I think you should remove it completely.

Reply:
The third argument is used for calculating whether more fields are to be fetched or not. Should I remove this? @seanlip

Reply:
Oh okay. This is not used. Got your point. Thanks!

Reply:
Done. Removed.